modules/n1ql/partials/n1ql-language-reference/transaction-restrictions.adoc

```diff
@@ -4,4 +4,4 @@
 
 * The `PREPARE` and `EXECUTE` statements are only permitted in a transaction for the DML statements listed above.
 
-All statements within a transaction are sent to the same Query node.
\ No newline at end of file
+// All statements within a transaction are sent to the same Query node.
```

Comment:
Why is this commented out?

Reply:
OK, may be worth either adding another explanatory comment, or simply deleting the line.

Reply:
Line has been removed